plugins/CoreHome/vue/src/createAngularJsAdapter.ts

```diff
@@ -134,7 +153,7 @@ export default function createAngularJsAdapter<InjectTypes = []>(options: {
                   rootVueTemplate += ` @${eventName}="onEventHandler('${eventName}', $event)"`;
                 }
               } else {
-                rootVueTemplate += ` :${info.vue}="${info.vue}"`;
+                rootVueTemplate += ` :${toKebabCase(info.vue)}="${info.vue}"`;
```

Comment:
Seemed to work w/o kebab case, but as far as I know it shouldn't, so changed it.